pkg/node/uevent_helpers.go

```diff
@@ -193,7 +193,7 @@ func updateDriveProperties(drive *directcsi.DirectCSIDrive, device *sys.Device)
 		updated = true
 	}
 
-	if drive.Status.Mountpoint != device.FirstMountPoint {
+	if device.FirstMountPoint != "" && drive.Status.Mountpoint != device.FirstMountPoint {
```

Comment:
Time being this check is OK. However we should update if `FSUUID` is updated above.